### DISCUSSION TITLE
Is clicking on the status bar supposed to do something

### DISCUSSION THREAD

**Discussion by frosencrantz** [saulpw/visidata#1616] [Q&A]
This code makes it look like clicking on the status bar will work: https://github.com/saulpw/visidata/blame/develop/visidata/statusbar.py#L131-L134

When I right-click on the statusbar it doesn't do anything:
https://asciinema.org/a/g41VmcTEkgXxVw0J7955ZHcEo

**Comment by saulpw** [comment]
Clicking on the sheet name itself (not the whole status bar) should do something: left-click to go to the sheets sheet, right click to rename the sheet.

**Comment by frosencrantz** [comment]
That doesn't work.  Here I load a datafile and click on the status bar and it causes the current column to change.  It doesn't go to the sheets sheet or ask for a new name.  You can't see the cursor.

https://asciinema.org/a/cwIvucYMi1VohpbPO9teKhX4p

Perhaps there is an off by one error
